src/components/SummaryCard/_mixins.scss

```diff
@@ -0,0 +1,160 @@
+////
+/// Summary card mixins.
+/// @group summary-card
+/// @copyright IBM Security 2019
+////
+
+@import '../../globals/index';
+
+@mixin summary-card {
+  /// Summary card header height.
+  /// @type Length
+  /// @access private
+  $header--height: carbon--mini-units(4);
+
+  /// Summary card body height.
+  /// @type Length
+  /// @access private
+  $body--height: carbon--mini-units(38);
+
+  /// Summary card footer height.
+  /// @type Length
+  /// @access private
+  $footer--height: carbon--mini-units(6);
+
+  position: relative;
+  display: flex;
+  min-width: carbon--mini-units(36);
+  flex-direction: column;
+  overflow: hidden;
+
+  &-body {
+    @include carbon--type-style($name: body-long-01);
+
+    // `hidden` because the ScrollGradient will handle overflow:
+    overflow: hidden;
+
+    display: flex;
+    height: $body--height;
+    width: 100%;
+    border-bottom: 1px solid $ui-02;
+    padding: $carbon--spacing-05;
+    margin: 0;
+  }
+
+  &-header {
+    height: $header--height;
+    display: flex;
+    padding: $carbon--spacing-05 $carbon--spacing-05 0 $carbon--spacing-05;
+
+    &__title {
+      @include carbon--type-style(caption-01);
+      color: $text-02;
+      flex-grow: 1;
+      margin: 0;
+      padding: 0;
+    }
+  }
+
+  &-footer {
+    position: relative;
+    display: flex;
+    background-color: $ui-01;
+  }
+
+  &-action {
+    // Required for truncation:
+    display: inline-block;
+
+    flex: 1;
+
+    &.#{$prefix}--btn--icon-only {
+      justify-content: center;
+      width: $footer--height;
+      flex: 0 0 auto;
+
+      &,
```

Comment:
Would you mind explaining what these rules are doing?

Reply:
According to the design spec these icon-only buttons need to have white icons, but default button styles set them to blue40. The overrides have to be aggressive in this case because the defaults set blue40 for hover via `currentColor` AND then for hover on the icon path. 😭